smartsim/error/errors.py

```diff
@@ -149,3 +149,11 @@ class UnproxyableStepError(TelemetryError):
 
 class SmartSimCLIActionCancelled(SmartSimError):
     """Raised when a `smart` CLI command is terminated"""
+
+
+class PreviewException(Exception):
```

Comment:
This seems super-specific to warrant another class that inherits direction from ``Exception``.  it seems like we should just use ``SSUnsupportedError`` or this should inherit from ``SSUnsupportedError``.

Reply:
Changed to ```class PreviewException(SSUnsupportedError):```